Mail/Helpers/AppAssembly.swift

```diff
@@ -61,6 +61,9 @@ enum ApplicationAssembly {
             Factory(type: InfomaniakNotifications.self) { _, _ in
                 InfomaniakNotifications(appGroup: AccountManager.appGroup)
             },
+            Factory(type: FeatureFlagsManageable.self) { _, _ in
```

Comment:
Now we have a separate assembly for the NotificationServiceExtension and the Share extension.
We should add the factory there too.

Reply:
The two classes used by the App and both share extension are `ApplicationAssembly` and `NotificationServiceAssembly`